extras/facade/src/main/java/org/prevayler/contrib/facade/TransactionType.java

```diff
@@ -116,6 +117,7 @@ private TransactionType(String p_name) {
         /**
          * @since 0_2
          */
+        @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
I have some theories about how to avoid these suppressions, but I think it's fine for now. The fact that you've gone beyond the core and into the extras and demos at all is quite commendable!